Standalone/PythonTests/Automated/benchmark_data_aggregator.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+from argparse import ArgumentParser
+import json
+from pathlib import Path
+import time
+import copy
+import subprocess
+import requests
+
+from ly_test_tools.mars.filebeat_client import FilebeatClient
+
+class BenchmarkDataAggregator:
+    def __init__(self, workspace, logger, test_suite):
+        self.build_dir = Path(workspace.paths.build_directory())
+        self.results_dir = Path(workspace.paths.project()) / 'user' / 'Scripts' / 'PerformanceBenchmarks'
+        self.test_suite = test_suite
+        self.logger = logger
+        self.filebeat_client = FilebeatClient(logger)
+
+    def _process_benchmark(self, benchmark_dir):
```

Comment:
here and throughout: can we get a brief docstring describing what each function does as well as what the params are expected to be? i.e. something similar to:
```
def _process_benchmark(self, benchmark_dir):
    """
    Processes benchmarks using benchmark_dir to find metadata json files that allow it to...etc...etc..
    :param benchmark_dir: path to the metadata json files and benchmark files
    :return: tuple with 3 elements: benchmark_metadata, frame_stats, pass_stats 
    """
```

Reply:
UPDATE: I can't resolve this but it's resolved now